src/scraper.py

```diff
@@ -0,0 +1,20 @@
+from flask import(Blueprint)
+from flask import send_from_directory
+
+bp = Blueprint('scraper', __name__)
+
+
+@bp.route('/')
+def index():
```

Comment:
The file name is misleading. This is a router file. Let's follow the solid. Router will define the routes.
Controller will control the BL (Business Logic)